network/shuffle/pom.xml

```diff
@@ -43,6 +43,17 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.fusesource.leveldbjni</groupId>
+      <artifactId>leveldbjni-all</artifactId>
+      <version>1.8</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
```

Comment:
err ... actually it looks like we don't have `jackson-annotations` listed anywhere else.  (Its excluded from a couple of other depenencies, but thats it.)  Do you think its worth including just here?  Should it be changed in the root pom?